### DISCUSSION TITLE
GitHub Education rejection

### DISCUSSION THREAD

**Discussion by ikvi63270** [community/community#143535] [GitHub Education]
### Hi everyone,

I am trying to join GitHub Education.
I got rejected from the automatic application processing.
I do not have a student ID, so I used my enrollment papers instead.
Problem is I can't upload the PDF, I have to take a picture with the webcam.
Which then becomes a picture of my screen with the required papers, as everything is online.
Everything is E-learning so I am not close to campus which seems to be what causes the issue.
As the only papers available to me with information about the classes being online is in my native language (danish).
And the quality obviously gets very bad when taking pictures of screens.
It seems silly to not have an option to upload the documentation instead of taking a picture with the webcam.
I hope somebody can help me out, thanks in advance.

**Comment by misterob0t** [comment]
Hi @ikvi63270, 

I don't know if it can help you but I did it yesterday, I first put my school certificate then I showed it on my webcam (with poor quality) and I think it's is everything. 
Normally he gives you instructions on what you should do, right ?

**Reply by ikvi63270** [reply]
As I mentioned above I do not have a student ID, so I am taking a picture of my enrollment papers instead.
All the information needed is there, it just doesn't accept it.  I have followed the instructions which is why I am now writing in here.
As Github does not accept help requests on their education program via their support portal.
The whole thing seems silly. And I can see several people have had the same issue as me. I was hoping somebody had found a workaround but it does not seem to be the case as some of the help tickets have been open for almost a year.

**Comment by misterob0t** [comment]
You must also have the same first and last name in your medical certificate and in your Billing information.

**Comment by lllHunterlllSV** [comment]
**Hello! 👋**

Thank you for sharing your experience, and I understand how frustrating it can be when the process doesn't go as expected. Let's try to work through this together:

It sounds like the issue is mostly with the **upload method** and the specific **format requirements**. Here are a few things to consider and try:

- 📄 **PDF Upload Issue**: While it seems you're unable to upload the PDF directly, GitHub Education typically expects **photo documentation** in a specific format (usually a clear photo taken with a webcam). However, in your case, it's understandable that using an actual photo of a screen isn't ideal due to quality issues.

- 🌍 **E-Learning and Language**: Since you’re enrolled in an e-learning program, the key here will likely be providing clear, readable documentation that confirms your student status. Even if your documents are in Danish, they should still be accepted as long as they show your name, course, and school details clearly. It might help to **explain your situation** in the application form, mentioning that you're studying online and don't have a student ID. Include details like your program’s name, institution, and your enrollment status.

- 📸 **Improving the Picture Quality**: If you have to use a photo, make sure the **lighting is good**, and the paper is clearly visible. Try to avoid reflections or shadows. Using a smartphone with a camera might give better quality than a webcam. You can also use a photo editing tool to crop and enhance the image to make the text clearer.

- 💬 **Contact GitHub Support**: If you're still having trouble, it might be worth contacting [GitHub Education Support](https://support.github.com/) to explain your specific situation. You could mention that the application system doesn’t accommodate online learning and the language barrier, and ask if there’s an alternative way to submit your documents.

I completely understand the challenges you're facing, and I hope this helps. Best of luck with your GitHub Education application, and I hope everything works out! 😊
